### PR TITLE
Increase max age for entity list endpoints

### LINKED ISSUE #9138: Compare V1 and V2 performance results and adjust CDN
### Problem

We need v2 to perform as well as V1. We need to compare the k6 and importer performance results for v1 and v2.

### Solution

- Adjust the Cache control headers and adjust them to a higher value to improve performance.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by mgoelswirlds**
**Description**:

Increase max age for entity list endpoints.

* Increases max age for entity list endpoints in order to improve performance for these endpoints.
* Adds a note for extended caching to the openapi.yaml

**Related issue(s)**:

Fixes #9138

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by jascks — DISMISSED** [review]
LGTM

File: hedera-mirror-rest/config/application.yml [hunk 1]
```diff
@@ -88,12 +88,14 @@ hedera:
...
+            /api/v1/accounts: { "cache-control": "public, max-age=10" }
             /api/v1/accounts/:idOrAliasOrEvmAddress:
               { "cache-control": "public, max-age=2" }
             /api/v1/accounts/:idOrAliasOrEvmAddress/rewards:
               { "cache-control": "public, max-age=60" }
             /api/v1/blocks/:hashOrNumber:
               { "cache-control": "public, max-age=600" }
+            /api/v1/contracts: { "cache-control": "public, max-age=10" }
             /api/v1/contracts/:contractId/results/:consensusTimestamp([0-9.]+):
               { "cache-control": "public, max-age=600" }
```
Comment (jascks) [inline hedera-mirror-rest/config/application.yml:100]: Do we also need to add `/api/v1/contracts/results`(#9245) and `/api/v1/contracts/results/logs` (#9244)?

Comment (mgoelswirlds) [reply hedera-mirror-rest/config/application.yml:100]: I am still contemplating that, since that might help the performance.

Comment (mgoelswirlds) [reply hedera-mirror-rest/config/application.yml:100]: Done.

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by xin-hedera — APPROVED** [review]
LGTM

**Review by jascks — APPROVED** [review]
LGTM

**Pull request merged**